R/verb-summarise.R

```diff
@@ -45,6 +69,20 @@ check_summarise_vars <- function(dots) {
   }
 }
 
+summarise_verbose <- function(.groups, .env) {
+  is.null(.groups) &&
+    # TODO not sure whether this is correct
+    is_reference(asNamespace("dbplyr"), topenv(.env)) &&
```

Comment:
I think it should be `is_reference(topenv(.env), global_env())` as in dplyr — the goal is to detect whether or not this is called directly by the user (in which case it will be somewhere beneath the global environment), or indirect via a package (in which is case it will be beneath a package namespace).

Reply:
ah, that makes sense. I just had to catch the environment earlier on to make it work.